docker-compose.yaml

```diff
@@ -80,7 +80,7 @@ services:
     environment:
       - AIRBYTE_ROLE=${AIRBYTE_ROLE:-}
       - AIRBYTE_VERSION=${VERSION}
-      - API_URL=${API_URL}
+      - API_URL=${API_URL:-}
```

Comment:
I am not a fan of having a the default hardcoded in the code. I would prefer if it failed